Crashlytics/Crashlytics/Controllers/FIRCLSRolloutsPersistenceManager.h

```diff
@@ -14,9 +14,14 @@
 
 #if SWIFT_PACKAGE
 @import FirebaseCrashlyticsSwift;
-#else  // Swift Package Manager
+#elif __has_include(<FirebaseCrashlytics/FirebaseCrashlytics-Swift.h>)
 #import <FirebaseCrashlytics/FirebaseCrashlytics-Swift.h>
-#endif  // CocoaPods
+#elif __has_include("FirebaseCrashlytics-Swift.h")
+// If frameworks are not available, fall back to importing the header as it
+// should be findable from a header search path pointing to the build
+// directory. See #12611 for more context.
+#import "FirebaseCrashlytics-Swift.h"
+#endif
```

Comment:
Originally, I had this in an `#else` clause, but that broke the Swift check imports CI that asserts that any `#else` clause following an `#IF SWIFT_PACKAGE` uses a framework-style import. So, I reverted back to this which appeases the check.

Now, there isn't an `#else` case to fallback on, but I don't think it matters because I was previously making the assumption that the "FirebaseCrashlytics-Swift.h" header would exist in `#else`  case.